services/catalog/tests/unit/with_dbs/test_entrypoint_services.py

```diff
@@ -376,14 +376,84 @@ async def test_get_service_resources(
     assert response.status_code == 200, f"{response.text}"
     data = response.json()
     received_resources = ServiceResources.parse_obj(data)
-    assert received_resources == expected_resources
+
+    expected_service_resources = ServiceResources.from_resources(
+        expected_resources, f"{service_key}:{service_version}"
+    )
+    assert received_resources.dict() == expected_service_resources.dict()
+    assert received_resources.json() == expected_service_resources.json()
+    assert received_resources == expected_service_resources
```

Comment:
MINOR: the last line suffice... the rest is "re-testing pydantic"

Reply:
let me format this properly and not be lazy.